docs/deb-rpm-repo-migration.md

```diff
@@ -0,0 +1,103 @@
+# DEB/RPM Repo Migration
+
+As part of our integration with Splunk, we are switching over our APT and RPM
+packages of the Smart Agent to a new repository (`splunk.jfrog.io`) and signing
+them with Splunk's key. The 5.0.0 release of the agent will be the first
+version of the Smart Agent that will exist only in the new repositories.
+
+All pre-5.0.0 versions of the Smart Agent will be mirrored over to the new
+`splunk.jfrog.io` repository and will be available from our original
+`dl.signalfx.com` repositories.
+
+Our old APT/Debian and RPM repositories at `dl.signalfx.com` will not be
+supported at all after April 21, 2020 and may cease to work at any point after
+that.
+
+## Migration Process
+
+### APT
+
+1. Delete the existing SignalFx apt key.
+
+```sh
+$ sudo apt-key del 5AE495F6
+```
+
+2. Import the new key:
+
+```sh
+$ curl https://splunk.jfrog.io/splunk/signalfx-agent-deb/splunk-B3CD4420.gpg | sudo apt-key add -
+```
+
+3. Replace the contents of /etc/apt/sources.list.d/signalfx-agent.list with:
+
+```sh
+deb https://splunk.jfrog.io/splunk/signalfx-agent-deb release main
+```
+
+4. Update the package metadata.
+
+```sh
+$ sudo apt-get update
+```
+
+5. Install the latest signalfx-agent.
+
+```sh
+$ sudo apt-get upgrade signalfx-agent
+```
+
+### RPM
+
+1. Delete the existing SignalFx RPM key.
+
+```sh
+$ rpm -e gpg-pubkey-098acf3b-55a5351a
+```
+
+2. Modify /etc/yum.repos.d/signalfx-agent.repo. Specifically: 
+
+ * Change baseurl to https://splunk.jfrog.io/splunk/signalfx-agent-rpm/release
+
+ * Change gpgkey to https://splunk.jfrog.io/splunk/signalfx-agent-rpm/splunk-B3CD4420.pub
+
+ The file should now look like:
+ ```sh
+ [signalfx-agent]
```

Comment:
Leading space?

Reply:
What about the equivalent for APT?

Reply:
> Leading space?

Will remove.

Reply:
> What about the equivalent for APT?

APT instructions: https://github.com/signalfx/signalfx-agent/pull/1221/files#diff-bfa2b6d288e4320f453c73ac7f5af90cR32